src/components/TodoContainer.js

```diff
@@ -1,140 +1,84 @@
-import React, { useState, useRef, useCallback } from "react";
```

Comment:
I think this is not a reusable component, can you move it outside of the component folder?